scripts/openstack-quickstart-demosetup

```diff
@@ -466,7 +470,7 @@ done
 ## setup loopback LVM configuration
 #-----------------------------------------
 
-
+start_and_enable_service lvm2-lvmetad
```

Comment:
that works for me as well :) I am not sure why we did the separation into two scripts, but I think it would logically belong there